opennms-doc/guide-admin/src/asciidoc/text/bsm/bsm-admin.adoc

```diff
@@ -0,0 +1,134 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../../images
+
+_OpenNMS_ provides a very powerful and fine grained service assurance layer based on the _Pollerd_ daemon.
+The service assurance layer monitors and tracks the status of nodes, interfaces and services.
```

Comment:
i.e.: a managed entity is an entity that is managed by an entity manager which manages all the managed entities.